deno.json

```diff
@@ -15,14 +15,20 @@
   "tasks": {
     "test": "deno test --allow-all --junit-path reports/junit.xml --coverage=reports/coverage/",
     "test-update": "deno task test -- --update",
-    "compile": "deno compile --output $OUTPUT_FILE_NAME --allow-env --allow-net=api.github.com --allow-run --allow-read=/tmp,/var/folders,/home/runner/work/_temp --allow-write=/tmp,/var/folders,/home/runner/work/_temp --target $DENO_TARGET index.ts"
+    "compile": "deno compile --output $OUTPUT_FILE_NAME --allow-env --allow-net=api.github.com --allow-run --allow-read=/tmp,/var/folders,/home/runner/work/_temp --allow-write=/tmp,/var/folders,/home/runner/work/_temp --target $DENO_TARGET index.ts",
+    "lint": "deno lint --fix"
   },
   "fmt": {
     "lineWidth": 150,
     "semiColons": false,
     "include": ["**/*.ts"],
     "exclude": ["**/*.json", "**/*.md", "**/*.yml", "**/*.yml", "**/*.lock"]
   },
+  "lint": {
```

Comment:
this is the only lint change I made. I do prefer the `async() => {}` syntax even when there are no `await` calls made in the function. The alternative is to modify the code blocks with `() => Promise.resolve(...)` and that seems like a log of work with no value.